pyaedt/application/Analysis.py

```diff
@@ -63,16 +64,12 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
 
     @property
     def materials(self):
-        """Material Manager that is used to manage materials in the project.
+        """Material manager for managing materials in the project.
         
-        Parameters
-        ----------
-
         Returns
         -------
         :class:`pyaedt.modules.MaterialLib.Materials`
-            Material Manager that is used to manage materials in the project.
-           
+            Material manager for managing materials in the project.     
```

Comment:
```suggestion
            Manages materials in the project.
```
Same comment as above. Feel free to ignore this.